### DISCUSSION TITLE
Wear os app

### DISCUSSION THREAD

**Discussion by Rexx-DZN** [dzikirqu/dzikirqu-android#35] [Ideas]
Hey man I have just seen this app by a random chance. I really am liking it so far. I would like the addition of a wear os companion app! With Salah tile and complications.

**Comment by wagyufari** [comment]
Assalamu'alaikum, thank you so much for your interest in our app. Unfortunately our development team is currently on a hiatus. We are planning on rebuilding the app from scratch to support the iOS Platform. But we will take that suggestion in our to-do list when creating the new app. 👍

**Reply by Rexx-DZN** [reply]
Salaam brother, does this mean that new features would be added to the new android app or so. If so how can I keep myself updated
